jsk_spot_robot/spoteus_demo/scripts/download_autowalk_data.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+from jsk_data import download_data
+
+
+def main():
+    PKG='spoteus_demo'
```

Comment:
Could you insert the spaces like `PKG = 'spoteus_demo` ?

Reply:
Fixed format